pallets/weights/src/polymesh_contracts.rs

```diff
@@ -242,12 +242,12 @@ impl polymesh_contracts::WeightInfo for WeightInfo {
     // Storage: Instance2Group InactiveMembers (r:1 w:0)
     // Storage: Identity Claims (r:2 w:0)
     fn dummy_contract() -> Weight {
-        (468_460_000 as Weight)
+        (282_229_000 as Weight)
             .saturating_add(DbWeight::get().reads(12 as Weight))
             .saturating_add(DbWeight::get().writes(2 as Weight))
     }
-    fn basic_runtime_call(_n: u32) -> Weight {
-        (1_440_000 as Weight)
+    fn basic_runtime_call(_n: u32, ) -> Weight {
+        (1_570_000 as Weight)
```

Comment:
Ignoring the `n` parameter is fine.  Most likely we don't even need the `n` parameter in the benchmark.  This weight function is to get the overhead of making a cheap runtime call.